src/main/java/org/folio/entitlement/integration/kafka/CapabilitiesModuleEventPublisher.java

```diff
@@ -38,13 +43,34 @@
 import org.folio.entitlement.integration.kafka.model.Endpoint;
 import org.folio.entitlement.integration.kafka.model.FolioResource;
 import org.folio.entitlement.integration.kafka.model.ModuleType;
+import org.folio.entitlement.integration.kafka.model.PermissionMappingValue;
 import org.springframework.stereotype.Component;
 
 @Log4j2
 @Component
 @RequiredArgsConstructor
 public class CapabilitiesModuleEventPublisher extends AbstractModuleEventPublisher<CapabilityEventPayload> {
 
+  private static Map<String, PermissionMappingValue> permissionMapping = Collections.emptyMap();
+
+  //  load permission mappings from a JSON file
+  static {
+    ObjectMapper objectMapper = new ObjectMapper();
+    try {
+      InputStream mappingFileAsStream = CapabilitiesModuleEventPublisher.class.getClassLoader()
+        .getResourceAsStream("permissionmappings/mapping.json");
+      permissionMapping = objectMapper.readValue(
+        mappingFileAsStream, new TypeReference<>() {
+        });
+    } catch (IOException e) {
+      log.error("Can't initialize Permission mapping", e);
+    }
+  }
+
+  public static PermissionMappingValue getValue(String key) {
```

Comment:
Does this logic really belong in CapabilitiesModuleEventPublisher? Seems like having a separate class that would be a Spring singleton bean would be more reusable and testable than a bunch of static methods.